src/mnode/src/mnodeAcct.c

```diff
@@ -215,7 +215,7 @@ static int32_t mnodeCreateRootAcct() {
   taosEncryptPass((uint8_t *)TSDB_DEFAULT_PASS, strlen(TSDB_DEFAULT_PASS), pAcct->pass);
   pAcct->cfg = (SAcctCfg){
     .maxUsers           = 128,
-    .maxDbs             = 128,
```

Comment:
Since the type of maxdbs field in the `log.acct_root` is SMALLINT, which is `i16`, it is to change the default value of .maxDbs as INT16_MAX here.